xcell/mappers/mapper_Planck_base.py

```diff
@@ -27,21 +27,14 @@ def _get_Planck_defaults(self, config):
         self.mask = None
         self.beam = None
         self.beam_info = None
-        self.gal_mask_mode = config.get('gal_mask_mode', '0.6')
-        self.gal_mask_modes = {'0.2': 0,
-                               '0.4': 1,
-                               '0.6': 2,
-                               '0.7': 3,
-                               '0.8': 4,
-                               '0.9': 5,
-                               '0.97': 6,
-                               '0.99': 7}
```

Comment:
Ok, you moved this to the individual mappers, but check my main comment for this review.

Reply:
All addressed, see other branch